README.md

```diff
@@ -68,11 +68,16 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
 
 ## Installation
 
-We offer three installation methods:
+We offer the following installation methods:
 
-1. **Running locally with “Docker for Desktop”** (~20 minutes) You will build
+1. **Running locally** (~20 minutes) You will build
    and deploy microservices images to a single-node Kubernetes cluster running
-   on your development machine.
+   on your development machine. There are two options to run a Kubernetes
+   cluster locally for this demo:
+   - [Minikube](https://github.com/kubernetes/minikube). Recommended for the
+     Linux hosts (also supports Mac/Windows).
+   - [Docker for Desktop](https://www.docker.com/products/docker-desktop).
+     Recommended for Mac/Windows.
 
 2. **Running on Google Kubernetes Engine (GKE)”** (~30 minutes) You will build,
```

Comment:
:wave: @idvoretskyi .  Note you didn't create this bug, however, for line 82 and others in a list, markdown just needs `1` and will autonumber ordered lists.

Reply:
@sdake oh, great to know :)